networkit/cpp/distance/SSSP.cpp

```diff
@@ -76,12 +76,15 @@ std::set<std::vector<node>> SSSP::getPaths(node t, bool forward) const {
 	return paths;
 }
 
-
-std::vector<node> SSSP::getStack(bool moveOut) {
+std::vector<node> SSSP::getDistanceVector(bool moveOut) {
 	if (!storeStack) {
 		throw std::runtime_error("stack has not been stored");
 	}
-	return (moveOut)?std::move(stack):stack;
+	return moveOut ? std::move(stack) : stack;
```

Comment:
What happens if I call sssp.getDistanceVector(true) and then sssp.getDistanceVector()? 

Should an exception been thrown if the vector has been moved out and the function is called again (with or without "true" as the parameter)

Reply:
Good point. Thoughts on newest changes there?